client/src/components/organisms/RequestGroupForm.tsx

```diff
@@ -0,0 +1,351 @@
+                inputComponent={<TextField
+                  name="name"
+                  placeholder="Enter a group name"
+                  type="text"
+                  input={name}
+                  isDisabled={false}
+                  isErroneous={nameError !== ""}
+                  onChange={onNameChange}
+                />
+                }
+              />
+            </div>
+            <div className="tag-input-form-item">
+              <FormItem
+                formItemName="Item Types"
+                instructions="If no types are applicable,  create a universal type such as “One Size”"
+                errorString={requestTypesError}
+                isDisabled={false}
+                tooltipText="Types describe more specific information about a request, such as size, capacity, or intended child age."
+                inputComponent={
+                  <TagInput
+                    tagStrings={requestTypeNames}
+                    placeholder="Enter a new type"
+                    actionString="Add new type:"
+                    isErroneous={requestTypesError !== ""}
+                    onChange={onInputRequestTypeNameChange}
+                    onSubmit={onAddRequestType}
+                    onDelete={onDeleteRequestType}
+                  />
+                }
+              />
+            </div>
+            <div className="richtext-field-form-item">
+              <FormItem
+                formItemName="Description & Requirements"
+                instructions="Formatting Tip: Ctrl-B to bold, “-” + Space to create a bullet point"
+                errorString={descriptionError}
+                isDisabled={false}
+                inputComponent={
+                  <RichTextField
+                    initialContent={initialRequestGroup && initialRequestGroup.description ? initialRequestGroup.description : ""}
+                    defaultText="Enter group description here"
+                    onChange={onDescriptionChange}
+                    isErroneous={descriptionError !== ""}
+                  />
+                }
+              />
+            </div>
+          </div>
+          <div className="request-group-form-modal-panel" id="right">
+            <div className="imagepicker-form-item">
+              <FormItem
+                formItemName="Image"
+                errorString={imageError}
+                isDisabled={false}
+                inputComponent={
+                  <ImagePicker
+                    onImageChange={onImageChange}
+                    images={images}
+                    selected={image}
+                    isErroneous={imageError !== ""}
+                  />
+                }
+              />
+            </div>
+          </div>
+        </div>
+        <div className="request-group-form-modal-footer">
+          <Button
+            text={props.operation === "create" ? "Create request group" : "Edit request group"}
```

Comment:
props.operation is a bit confusing like this. Perhaps set the text outside of the UI and specifically specify which operation produces which text (e.g. switch case)

Reply:
will do!